ui/pages/Report/components/DiscoverySheet.jsx

```diff
@@ -1,30 +1,62 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+import { NavLink } from 'react-router-dom'
 
+import AwesomeBar from 'shared/components/page/AwesomeBar'
+import DataTable from 'shared/components/table/DataTable'
+import DataLoader from 'shared/components/DataLoader'
+import { InlineHeader } from 'shared/components/StyledComponents'
 import { DISCOVERY_SHEET_COLUMNS, CMG_PROJECT_PATH } from '../constants'
 import { loadDiscoverySheet } from '../reducers'
 import { getDiscoverySheetLoading, getDiscoverySheetLoadingError, getDiscoverySheetRows } from '../selectors'
-import BaseReport from './BaseReport'
```

Comment:
BaseReport was only used in this file and in SampleMetdata, and since I completely reconfigured that second usage and this is going to be deprecated soon I decided to roll the functionality up into this file directly instead of maintaining it as a separate helper  class